packages/world/ts/objects.ts

```diff
@@ -384,16 +378,17 @@ export const categoryObjects: {
     { name: "RedSand", mass: 5000000000000000n },
     { name: "Sandstone", mass: 30000000000000000n },
     { name: "RedSandstone", mass: 37500000000000000n },
-  ],
-  Clay: [
     { name: "Clay", mass: 2400000000000000n },
-    { name: "Terracotta", mass: 18000000000000000n },
-    { name: "BrownTerracotta", mass: 22500000000000000n },
-    { name: "OrangeTerracotta", mass: 30000000000000000n },
-    { name: "WhiteTerracotta", mass: 22500000000000000n },
-    { name: "LightGrayTerracotta", mass: 30000000000000000n },
-    { name: "YellowTerracotta", mass: 30000000000000000n },
-    { name: "RedTerracotta", mass: 30000000000000000n },
+  ],
+  Terracotta: [
```

Comment:
I moved `Clay` into `Soil` category since I added `AnyTerracotta` and it needs to be the first one in the category. I aligned the masses to be the same as likely with all the different variants a recipe will require `AnyTerracotta`